src/StatusQ/Controls/StatusInput.qml

```diff
@@ -419,13 +423,13 @@ Item {
 
         StatusBaseText {
             id: errorMessage
-            visible: !!text && !statusBaseInput.valid
-            height: visible ? contentHeight : 0
+            visible: !statusBaseInput.valid && statusBaseInput.dirty && !!text
```

Comment:
This depends on the type of validation. The always mode doesn't depend on the dirty state

Reply:
Ok, i'll add this check

Reply:
<img width="623" alt="Screenshot 2022-08-05 at 14 01 14" src="https://user-images.githubusercontent.com/2522130/183064258-77dcb394-c985-4ec0-8be9-49f319c874d3.png">
 @jrainville for me it's a bit strange that input borders are blue when `ValidationMode.Always`

Reply:
I think it's blue in this case because we are focusing on it? If you click on another input, does it show red?

Reply:
No, border is defined by StatusBaseInput and it's not aware of ValidationMode. Imho we should managed valid flag in StatusInput and move there logic from StatusBaseInput